src/theme/theme.js

```diff
@@ -1,12 +1,593 @@
+/* eslint-disable no-underscore-dangle */
 // The default theme.js for Paragon. The structure of this object follows the
 // theme specification found here: https://system-ui.com/theme/. We prefer to add
 // to this theme over time rather than all valid keys at once to ensure everything
 // present here is used in the system.
 
+// Color system
+const colors = {
+  white: '#fff',
+  gray100: '#ebebeb',
+  gray200: '#cccccc',
+  gray300: '#adadad',
+  gray400: '#8f8f8f',
+  gray500: '#707070',
+  gray600: '#5c5c5c',
+  gray700: '#454545',
+  gray800: '#333333',
+  gray900: '#212529',
+  black: '#000',
+
+  blue: '#23419f',
+  red: '#C32D3A',
+  yellow: '#ffd900',
+  green: '#178253',
+  teal: '#006DAA',
+};
+colors.gray = colors.gray500;
+colors.darkGray = colors.gray700;
+
+const themeColors = {
+  primary: '#0A3055',
+  secondary: colors.gray700,
+  brand: '#9D0054',
+  success: colors.green,
+  info: colors.teal,
+  danger: colors.red,
+  warning: colors.yellow,
+  light: '#E1DDDB',
+  gray: colors.gray,
+  dark: '#273F2F',
+  accentA: '#00BBF9',
+  accentB: '#FFEE88',
+};
+
+const _mix = (c1, c2, perc) => `mix(${c1}, ${c2}, ${perc}%`;
+const _blackMix = (color, perc) => _mix(
+  colors.black, colors[color], perc,
+);
+const _whiteMix = (color, perc) => _mix(
+  colors.white, colors[color], perc,
+);
+const genColorLevels = (colorKey) => ({
```

Comment:
Does this produce the same hex values for all the colors as the original theme?  (I'm not familiar with how it generates colors)